pkg/block/block_linux.go

```diff
@@ -277,6 +277,9 @@ func disks(ctx *context.Context, paths *linuxpath.Paths) []*Disk {
 		size := diskSizeBytes(paths, dname)
 		pbs := diskPhysicalBlockSizeBytes(paths, dname)
 		busPath := diskBusPath(paths, dname)
+		if strings.Contains(busPath, "-iscsi-") {
+			driveType = DRIVE_TYPE_ISCSI
```

Comment:
I wonder if there is another way to know if a device is iscsi or not

Reply:
Not that I know of.  This method has worked consistently for me.

Reply:
We can surely start like this